### ISSUE TITLE
add python information to HPC software guide

### ISSUE THREAD

**Issue by ChristinaLK** [CHTC/chtc-website-source#149]

**Comment by jmvera255** [comment]
the first thing should be whether they intend to use python for MPI work, otherwise they should reach out to us about using the HTC or for performing the work locally

**Comment by ChristinaLK** [comment]
True. I think some people use python to set up submit files + do other "administrative" type tasks, so I don't think we should limit it to purely MPI work. But we can have a disclaimer/list of use cases.
